mne/channels/channels.py

```diff
@@ -932,10 +932,14 @@ def read_ch_connectivity(fname, picks=None):
 
     Returns
     -------
-    ch_connectivity : scipy.sparse matrix
+    ch_connectivity : scipy.sparse matrix, shape (n_channels, n_channels)
         The connectivity matrix.
     ch_names : list
         The list of channel names present in connectivity matrix.
+
+    See Also
+    --------
+    find_ch_connectivity
```

Comment:
Somewhere -- probably in a Notes section here -- we should say how these two functions are related. Perhaps we should say that `find_ch_connectivity` is generally preferred as it's more general?